bin/fin

```diff
@@ -2844,6 +2844,30 @@ ssh_add ()
 	# Only do this when given an argument (key name) or nothing, but not for options (e.g. -l)
 	[[ "${1#-}" == "$1" ]] && docker cp "$ssh_path" docksal-ssh-agent:/
 
+	if [[ "${1#-}" == "$1" ]]; then
```

Comment:
We should improve this check. A key can have a dashes in it's name (`acquia-key`) and this condition removes dashes in a string and compares to its original will be false.

```bash
if [[ ! "${1}" =~ ^- ]]; then
```